src/test/checksum/file1

```diff
@@ -1,2 +1,3 @@
 Hello.  This is a simple text file with a known checksum.
 The checksum unit test is run on this file to check for regressions.
+
```

Comment:
I added adequate ASSERTs which check the size. I'll explain this change in the commit message.